src/widgets/helper/ResizingTextEdit.cpp

```diff
@@ -270,7 +275,53 @@ bool ResizingTextEdit::canInsertFromMimeData(const QMimeData *source) const
 
 void ResizingTextEdit::insertFromMimeData(const QMimeData *source)
 {
-    if (!source->hasImage())
+    if (source->hasImage())
+    {
+#if 0
+        static QUrl url("http://localhost:7494/upload?password=xd");
+        // default port and password for nuuls' filehost.
+#else
+        static QUrl url("https://i.nuuls.com/upload");
+        // production site
+#endif
+
+        QImage image = qvariant_cast<QImage>(source->imageData());
+
+        const char *boundary = "thisistheboudaryasd";
+        QByteArray dataToSend;
+        dataToSend.insert(0, "--");
+        dataToSend.append(boundary);
+        dataToSend.append(
+            "\r\n"
+            "Content-Disposition: form-data; name=\"attachment\"; "
+            "filename=\"control_v.png\"\r\n"
+            "Content-Type: image/png\r\n"
+            "\r\n");
```

Comment:
is there really no way to do a multipart file upload with this library? This seems hackish to me.

Reply:
I don't know other ways I tried were even worse and didn't work LULW